src/assets/styles/doc-styles.scss

```
@@ -0,0 +1,822 @@
+    }
+
+    .dark & .small-circle {
+      transform: translate(-20px, 10px);
+      transform: scale(0);
+    }
+
+    .dark & .sun-rays {
+      transform: scale(1);
+    }
+  }
+}
+
+.visually-hidden {
+  clip: rect(0 0 0 0);
+  clip-path: inset(50%);
+  height: 1px;
+  overflow: hidden;
+  white-space: nowrap;
+  width: 1px;
+}
+
+.header-anchor {
+  position: absolute;
+  transform: translate(-99%, -100%);
+  box-sizing: border-box;
+  @apply pr-2 opacity-0 transition;
+}
+
+.header-anchor:hover,
+:hover + .header-anchor {
+  @apply opacity-100;
+}
+
+.parcel-docs-header {
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1;
+
+  display: grid;
+  grid-template-areas: "logo picker . search social";
+  grid-template-columns: auto auto 1fr auto auto;
+  align-items: center;
+  box-sizing: border-box;
+
+  @apply bg-blueGray-700;
+  @include transition-background-color;
+
+  padding: 1rem;
+  color: #e7dacb;
+  width: 100%;
+
+  .left {
+    grid-area: logo;
+    display: flex;
+    .hamburger-button {
+      @apply w-6 h-6 mr-4 flex-shrink-0 lg:hidden;
+    }
+    
+    .logo {
+      @apply sm:mr-16;
+      display: flex;
+      align-items: center;
+      height: 30px;
+      flex-shrink: 0;
+
+      .parcel-text {
+        width: 120px;
+        @apply hidden sm:block;
+      }
+
+      .parcel {
+        margin-right: 10px;
+      }
+    }
+  }
+
+  .search-container {
+    position: relative;
+    grid-area: search;
+    svg {
+      @apply h-6 w-6 md:h-5 md:w-5;
+      position: absolute;
+      top: 6px;
+      left: 8px;
+      z-index: 1;
+      pointer-events: none;
+
+      @media (min-width: 768px) {
+        top: 7px;
+      }
+    }
+
+    input {
+      @apply px-5 md:pl-8 focus:pl-8 py-1 bg-blueGray-700 border border-transparent;
+      @apply md:rounded-full md:bg-blueGray-800 md:bg-opacity-70 md:border-blueGray-600;
+      @apply focus:rounded-full focus:bg-blueGray-800 focus:bg-opacity-70;
+      @apply w-0 focus:w-48 md:w-48 transition-all;
+  
+      &:focus {
+        outline: none;
+        @apply ring ring-offset-0 ring-sky-700 border-sky-600;
+      }
+    }
+  }
+
+  &.search .search-container svg {
+    @apply h-5 w-5;
+    top: 7px;
+    @media (max-width: 768px) {
+      top: 8px;
+    }
+  }
+
+  .section-picker {
+    grid-area: picker;
+    display: flex;
+    justify-content: center;
+    position: absolute;
+
+    @apply rounded-full bg-blueGray-800 transition;
+
+    a {
+      display: block;
+
+      @apply transition text-blueGray-400 hover:text-blueGray-200 rounded-full m-1 px-3 sm:px-5 py-1;
+      &.selected {
+        @apply bg-sky-900 text-gray-100;
+      }
+    }
+  }
+
+  .social-links {
+    grid-area: social;
+    display: flex;
+    justify-content: flex-end;
+    padding-left: 1rem;
+    white-space: nowrap;
+
+    button, a {
+      color: inherit;
+      @apply transition-colors;
+
+      &:hover {
+        color: lighten(#e7dacb, 10%);
+      }
+    }
+
+    svg {
```

```suggestion
    a svg {
```

Changing this should fix the dark mode switcher color.

svgo removes `fill="black"` (which seems to be the default), but the `svg` also has `fill: currentColor`, which overrides the previously implicit `fill="black"`